cmd/livepeer/starter/starter_test.go

```diff
@@ -99,3 +102,66 @@ func TestParseGetBroadcasterPrices(t *testing.T) {
 	assert.Equal(big.NewRat(1000, 1), price1)
 	assert.Equal(big.NewRat(2000, 3), price2)
 }
+
+func TestFlag_KeystorePath_File(t *testing.T) {
+	assert := assert.New(t)
+
+	var addr = "0x0000000000000000000000000000000000000001"
+	var fname = "UTC--2023-01-05T00-46-15.503776013Z--" + addr
+	tempDir := t.TempDir()
+	file1, err := ioutil.TempFile(tempDir, fname)
```

Comment:
```suggestion
	file1, err := os.CreateTemp("", "")
```

`ioutil.TempFile()` is deprecated